### DISCUSSION TITLE
Table of Content : how to hide/customize the added permalink icon ?

### DISCUSSION THREAD

**Discussion by leobenkel** [skeletonlabs/skeleton#1847] [Q&A] [answered]
Hello,

<img width="35" alt="Screenshot 2023-08-11 at 13 14 24" src="https://github.com/skeletonlabs/skeleton/assets/4960573/7d029cf1-3a8a-44e7-8b2a-517ddd0612c8">

How do I hide/modify this icon? it doesnt really fit with my design and I would like it removed or at least modified to it looks better. Thanks !

**Comment by Sarenor** [comment, marked as answer]
Table of Content is currently undergoing a huge refactor and will be released together with the rest of Skeleton v2.
The release of a release candidate for testing for v2 is planned for the coming Tuesday (so the 15th of August).

You can take a look at the new version here: https://skeleton-docs-git-v2-skeleton-labs.vercel.app/utilities/table-of-contents

**Reply by endigo9740** [reply]
Adding one note - this will remove the link icon shown. Which means it will solve the issue raised here!

**Reply by leobenkel** [reply]
Awesome ! Thank you ! Looking forward to it <3
